README.md

```diff
@@ -85,3 +85,11 @@ Ansonsten können Formeln in ihrer vollen Pracht nur in den PDFs betrachtet werd
 
 Zusätzlich zu den PDFs generieren wir zu allen Modulen Anki-Decks.
 Damit diese möglichst problemlos generiert werden können, sollten auch die Markdown Extensions [hier](https://github.com/Steve2955/md2apkg) beachtet werden.
+
+Syntax von `anki.list`: Zeilenweise. Pfad zu Datei angeben. Semikolon separiert Namen. Newline am Ende lassen.
```

Comment:
```suggestion
Syntax von `anki.list`: Zeilenweise. Pfad zu Datei angeben. Semikolon separiert Eingabe-Pfad von Output-Pfad. Newline am Ende lassen.
```

`anki.list` hat aber doch keine Newline am Ende 🤔

Reply:
Das EOF darf nicht in der gleichen Zeile stehen wie der letzte Pfad.
Also 
`./PRO-STRASS/README.md;PRO-STRASS.apkg\n`
und nicht ohne \n